### PR TITLE
Add `langchain.rb` to the Frameworks list

### PR DESCRIPTION
**Pull request by andreibondarev**
I want to contribute to this list and help the maintainers, so:

- [x] I've accepted the terms of the `CC0` License;
- [x] I've added the topic `rubyml` to the contributed repository if appropriate.

### CODE REVIEW

**Comment by andreibondarev** [comment]
@arbox For your review :)

File: readme.md [hunk 1]
```diff
@@ -94,6 +94,8 @@ programming languages with appropriate bindings for Ruby.
 
 ### Frameworks
 
+- [LangChain.rb](https://github.com/andreibondarev/langchainrb) -
+  Build ML/AI-supercharged applications with Ruby's LangChain
```
Comment (arbox) [inline readme.md]: Please add the trailing dot.
Any specific reason why this post should stay at the beginning of the list, @andreibondarev ?

Comment (andreibondarev) [reply readme.md]: Nope, but I didn't see any specific ordering in place.

**Comment by andreibondarev** [comment]
@arbox For your re-review :)

**Pull request merged**